prime-router/src/main/resources/db/migration/V43__item_hash.sql

```diff
@@ -0,0 +1,12 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+ALTER TABLE item_lineage
```

Comment:
I'll do more research, but I believe adding a column with a default like this is going to take quite some time (I think production item lineage is ~52m rows), and will lock the table while it does so.

Do you already have a process for generating the hash for all the existing rows? If so, one approach might be:

1. adding the column allowing nulls
2. backfill the hash for each row (may take a while but AFAIK these records are write-once so we shouldn't have any contention) while still allowing new records to be stored with the row
3. add the index (still concurrently)
4. maybe (?) mark the column as not null; if that takes too long (because it has to scan every record to ensure it's not null) we may be able to get away with a check constraint instead

Reply:
I can't speak to whether this will be slow or not, but a default of NULL for old rows seems fine to me.  It is neither possible nor very useful to backfill old rows.  I may be missing something, but it also seems ok to allow it to just stay as nullable even going forward.

Reply:
(will post more research in PR-level comments...)